### PR TITLE
Migrated the license to MIT

### LINKED ISSUE #56: License consideration?
Hi, I love all your controls here.
However, your license, the CPOL, prevents the library from being used in many open source projects because of its restrictions on "illegal" and "immoral" usage.
Could you please re-release this package under another popular license out there, such as the LGPLv3 or the MIT license? This could help other projects a lot!
Anyway, I don't see much use of notify icons for misdeeds anyway.

Thanks.

### PR DESCRIPTION
**Pull request by Lakritzator**
Also removed .NET 5.0 support as this is EOL.

Closes #56

### CODE REVIEW

**Comment by Lakritzator** [comment]
Created this for #56

**Comment by Lakritzator** [comment]
I also bumped up the version to 2.0.x as I consider a license change a potential breaking / major change.
Look at this to see how the file header was changed: https://github.com/hardcodet/wpf-notifyicon/pull/84/files#diff-ffd2fa4db00e15fe03e88dff6cc543d6be7000eeb99c57000dbfbdd44e90d889

**Comment by Lakritzator** [comment]
Oh, forgot, I also change the namespace to use file scoped namespace. I probably regret that change, not for the change itself but as some PR will no longer merge without manual work. 🙄

**Comment by trungnt2910** [comment]
It's been a while, what's the status of this PR?
I also see that this branch has conflicts with the base too.

**Comment by Lakritzator** [comment]
Life came in between, just wait.

**Comment by Lakritzator** [comment]
Hey @punker76 & @hardcodet can you take a look at this, so we can merge it?

**Comment by tstavropoulos** [comment]
Hey @Lakritzator and @hardcodet 

I saw the discussion about this in #56. This is a really great project, and pulling in this license change would definitely make my life easier. Any chance of that happening?  Thanks, and good luck!

File: LICENSE [hunk 1]
```diff
@@ -1,57 +1,23 @@
-The Code Project Open License (CPOL) 1.02
+The MIT License (MIT)
 
-Preamble
+Copyright (c) Dapplo and Contributors
```
Comment (hardcodet) [inline LICENSE:3]: This seems to have been copy/pasted from Dapplo ;)

Comment (Lakritzator) [reply LICENSE:3]: Damn, I was hoping you would over see it, so I have all rights... no just kidding.
Let me correct that...

**Comment by Linx-ESP** [comment]
Any news?

**Comment by kjean667** [comment]
I'm a bit interested in this license change as well. Are there any stoppers from this being merged? @hardcodet @punker76

**Comment by punker76** [comment]
@Lakritzator @hardcodet LGTM

**Comment by jeroenwalter** [comment]
Hi, any chance that this PR is added to the main base instead of waiting on a 2.0.0 milestone, that has no clear ETA? 
I found this fork, [https://github.com/HavenDV/H.NotifyIcon](https://github.com/HavenDV/H.NotifyIcon) , that changed the license to MIT, but I don't one can't just change the license without the express permission of the current maintainers.

**Pull request merged**